### DISCUSSION TITLE
solderless case for olimex ESP32-POE-ISO and EBYTE NRF24L01+

### DISCUSSION THREAD

**Discussion by raccoon0815** [tbnobody/OpenDTU#393] [Show and tell]
Hi, I have designed a solderless 3d printed case for the olimex ESP32-POE-ISO and the EBYTE NRF24L01+ (E01-ML01DP5) board.
So every part can be recycled again in the future. 

olimex ESP32-POE-ISO Version 
[https://www.thingiverse.com/thing:5661780](https://www.thingiverse.com/thing:5661780)

generic esp32dev Version
[https://www.thingiverse.com/thing:5632374](https://www.thingiverse.com/thing:5632374)

have fun :-)

**Comment by darkl0rdchen** [comment]
wow cool case!
one Vesion with a waveshare 1.54" e-ink display would also be great, so you can read the values ​​straight away

**Reply by raccoon0815** [reply]
yes would be a nice idea - The resources on the esp32 are limited... I would put the display on a raspberry pi zero to show other measured values ​​there as well.
